spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -561,7 +561,7 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
    * @param predictKey key to store predicted result
    * @return
    */
-  def predictImage(imageFrame: ImageFrame,
+  def predictImage(imageFrame: DistributedImageFrame,
```

Comment:
why not using just `predictImage` for both local and distributed and dispatching it to different handle according to ImageFrame type ? this would reduce the API exposed to users